### PR TITLE
Revision of interpolated strings topic.

### PR DESCRIPTION
**Pull request by rpetrusha**
# Revision of interpolated strings topic.

## Summary

In the SEO session last Monday, I started to convert this to a C#-only topic. I've also added some external code.

@stevehoag, I'd sent you an older version of the topic to create a VB equivalent. 

## Suggested Reviewers

@billwagner @stevehoag

### CODE REVIEW

File: docs/csharp/language-reference/keywords/interpolated-strings.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Interpolated Strings (C# and Visual Basic Reference) | Microsoft Docs"
+title: "Interpolated Strings (C#) | Microsoft Docs"
 ms.date: "2015-07-20"
```
Comment (BillWagner) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: You should update the date, based on the updates you made.

**Review by BillWagner — APPROVED** [review]
One comment to update the date. Otherwise, this looks great. :shipit:

File: docs/csharp/language-reference/keywords/interpolated-strings.md [hunk 2]
```diff
@@ -27,88 +27,68 @@ translation.priority.mt:
...
+
+ You can use an interpolated string anywhere you can use a string literal.  The interpolated string is evaluated each time the code with the interpolated string executes. This allows you to separate the definition and evaluation of an interpolated string.  
+  
+ To include a curly brace ("{" or "}") in an interpolated string, use two curly braces, "{{" or "}}".  See the Implicit Conversions section for more details.  
+
+If the interpolated string contains other characters with special meaning in an interpolated string, such as the quotation mark ("), colon (:), or comma (,), they should be escaped if they occur in literal text, or they should be included in an expression delimited by parentheses if they are language elements included in a interpolated expression. The following example escapes quotation marks to include them in the result string, and it uses parentheses to delimit the expression `(age == 1 ? "" : "s")` so that the colon is not interpreted as beginning a format string.
```
Comment (mairaw) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: a interpolated -> an interpolated

Comment (rpetrusha) [inline docs/csharp/language-reference/keywords/interpolated-strings.md]: @mairaw Fixed.

**Pull request merged**